src/DDTrace/Tracer.php

```diff
@@ -438,12 +416,12 @@ private function addUrlAsResourceNameToRootSpan()
             return;
         }
         // Normalized URL as the resource name
-        $normalizer = new Urls(explode(',', getenv('DD_TRACE_RESOURCE_URI_MAPPING')));
-        $span->setTag(
-            Tag::RESOURCE_NAME,
-            $_SERVER['REQUEST_METHOD'] . ' ' . $normalizer->normalize($_SERVER['REQUEST_URI']),
-            true
-        );
+        $resourceName = $_SERVER['REQUEST_METHOD'];
```

Comment:
Could we add an `isset()` check here to avoid something like #637?

Reply:
good idea